editor/editor.pro

```diff
@@ -0,0 +1,17 @@
+# Editor specific things.
+
+TARGET = editor
+TEMPLATE = lib
+CONFIG += staticlib warn_on
+INCLUDEPATH += ../3party/opening_hours/src
```

Comment:
What is the reason to use INCLUDEPATH? Everywhere in the code we include like this:
```
#include "3party/opening_hours/src/some_header.hpp"
```